code/game/objects/items/weapons/tools/simple_weapons.dm

```diff
@@ -464,6 +464,24 @@
 		slot_back_str = "renderslayer"
 		)
 
+/obj/item/tool/knife/ritual/blade
+	name = "awakened blade"
+	desc = "A sharp blade whose latent powers have fully awoken by the crayons' magic. \
```

Comment:
can this be wrote differently?
It just dosnt feeeel crayon magic

Reply:
How could I make this more crayon-y then?